src/Infrastructure/Utilities/CertificateUtility.cs

```diff
@@ -8,12 +8,12 @@ namespace Contoso.FraudProtection.Infrastructure.Utilities
 {
     public static class CertificateUtility
     {
-        public static X509Certificate2 GetCertificateBy<T>(X509FindType findType, T findValue, bool findValidOnly = true)
+        public static X509Certificate2 GetCertificateBy<T>(X509FindType findType, T findValue)
         {
             var certStore = new X509Store(StoreLocation.CurrentUser);
             certStore.Open(OpenFlags.ReadOnly);
 
-            X509Certificate2Collection certs = certStore.Certificates.Find(findType, findValue, findValidOnly);
+            X509Certificate2Collection certs = certStore.Certificates.Find(findType, findValue, false);
```

Comment:
if you remove the flag it should be true,not false -w e would like only valid certs to be returned.

Reply:
I think we want both so that a user can use a self-signed cert with testing.  Otherwise, self-signed certs aren't returned by this call.